test/system/data_entry_test.rb

```diff
@@ -58,7 +56,7 @@ class DataEntryTest < ActionDispatch::IntegrationTest
         assert_equal '2', lmp_days.value
         assert has_text? "Called on: #{2.days.ago.strftime('%m/%d/%Y')}"
         assert has_field?('Appointment date',
-                          with: 1.day.ago.strftime('%Y-%m-%d')) # TODO: Driver format problem
+                          with: 1.day.ago.strftime('%m/%d/%Y'))
```

Comment:
Something about this comparison is making it choke which is exceptionally weird.

Reply:
Turns out chrome stores the value of date fields as YYYY-MM-DD, which is really annoying! But easily fixable. Resolved in 608c919